.changeset/lovely-pianos-breathe.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
+---
+
+feat: custom file formats in astro/loader/file.ts
```

Comment:
It would be great if you could make a start with some basic docs in here too. This is the content that would go into the changelog, so that would be the right sort of format.

Reply:
Just added this! lmk if its too much